File: sdk/dotnet-sdk/src/Layotto/Configuration/ConfigurationRequestItem.cs
using System.Collections.Generic;

namespace Layotto.Configuration
{
    /// <summary>
    /// used for GET,DEL,SUB request
    /// </summary>
    public class ConfigurationRequestItem
    {
        /// <summary>
        /// The name of configuration store.
        /// </summary>
        public string StoreName { get; set; }

        /// <summary>
        /// The application id which
        /// Only used for admin, Ignored and reset for normal client
        /// </summary>
        public string AppId { get; set; }

        /// <summary>
        /// The group of keys.
        /// </summary>
        public string Group { get; set; }

        /// <summary>
        /// The label for keys.
        /// </summary>
        public string Label { get; set; }

        /// <summary>
        /// The keys to get.
        /// </summary>
        public List<string> Keys { get; set; }

        /// <summary>
        /// The metadata which will be sent to configuration store components.
        /// </summary>
        /// <returns></returns>
        public Dictionary<string, string> Metadata { get; set; }
    }
}

File: sdk/java-sdk/src/main/java/spec/sdk/runtime/v1/domain/core/configuration/ConfigurationRequestItem.java
package spec.sdk.runtime.v1.domain.core.configuration;

import java.util.List;
import java.util.Map;

/**
 * ConfigurationRequestItem used for GET,DEL,SUB request
 */
public class ConfigurationRequestItem {

    /**
     * The name of configuration store.
     */
    private String storeName;
    /**
     * The application id which
     * Only used for admin, Ignored and reset for normal client
     */
    private String appId;
    /**
     * The group of keys.
     */
    private String group;
    /**
     * The label for keys.
     */
    private String label;
    /**
     * The keys to get.
     */
    private List<String> keys;
    /**
     * The metadata which will be sent to configuration store components.
     */
    private Map<String, String> metadata;

    public String getStoreName() {
        return storeName;
    }

    public void setStoreName(String storeName) {
        this.storeName = storeName;
    }

    public String getAppId() {
        return appId;
    }

    public void setAppId(String appId) {
        this.appId = appId;
    }

    public String getGroup() {
        return group;
    }

    public void setGroup(String group) {
        this.group = group;
    }

    public String getLabel() {
        return label;
    }

    public void setLabel(String label) {
        this.label = label;
    }

    public List<String> getKeys() {
        return keys;
    }

    public void setKeys(List<String> keys) {
        this.keys = keys;
    }

    public Map<String, String> getMetadata() {
        return metadata;
    }

    public void setMetadata(Map<String, String> metadata) {
        this.metadata = metadata;
    }
}
